### PR TITLE
Add ZigStar BT Proxy Link

### PR DESCRIPTION
**Pull request by mercenaruss**
Add ZigStar Ethernet BT Proxy,what include ligh,temperature & humidity sensor.

### CODE REVIEW

**Comment by balloob** [comment]
I've never heard of this and have no clue if this is legit.

**Comment by mercenaruss** [comment]
> I've never heard of this and have no clue if this is legit.

I'm in DIY devices ,especially gateways for Zigbee,you can check : zig-star.com & https://community.home-assistant.io/t/zigstar-zigbee-coordinators-and-routers/338586
It's the same like TubeZS BT Proxy : https://zig-star.com/esp/bt-proxy/ , https://www.tindie.com/products/zigstar/zigstar-bt-proxy/
I just released the device,soon will release gerbers and self assembly files.

File: static/index.html [hunk 1]
```diff
@@ -327,6 +327,12 @@ <h3>Wireless-Tag WT32-ETH01</h3>
               href="https://www.tubeszb.com/product/wt32-eth01-bluetooth-proxy-kit/53"
               >TubesZB - Kit with enlosure and USB-C</a
             >
+          </li>
+		  <li>
+            <a
+              href="https://zig-star.com/esp/bt-proxy/"
+              >ZigStar BT Proxy with Temp/Humidy & Light Sensor</a
```
[the remark this answers is not in the document]
Comment (mercenaruss) [reply static/index.html:334]: Okay,i get it.
I provide separate .yaml for this option.
Will modify link description to show only BT Proxy functionality.

[the remark this answers is not in the document]
Comment (balloob) [reply static/index.html]: I suggest you start with creating your own copy of this website to offer this feature. It's open source https://github.com/esphome/bluetooth-proxies

**Pull request closed without merging**